### PR TITLE
task1 is completed

### PR DESCRIPTION
**Pull request by Oleksii-Fedorchuk**

### CODE REVIEW

File: Oleksii_Fedorchuk/HW14_Oleksii_Fedorchuk/task1.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+    def school_bio(self):
+        return f"School number is: {self.__school_number}\n" \
+               f"Director of the schools is: {self.__director}\n" \
+               f"Secretary of the school is: {self.__secretary}\n" \
+               f"Total number of students in school is: {self.__number_of_students}\n"
+
+
+class Classes:
+    """ Polymorphism"""
```
Comment (MarinaZelenska) [inline Oleksii_Fedorchuk/HW14_Oleksii_Fedorchuk/task1.py]: Could you please write comments where
Inheritance, # hiding, # incapsulation, how it is written in homework. In your homework, I can't see that you understand this principles

File: Oleksii_Fedorchuk/HW14_Oleksii_Fedorchuk/task1.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        self.__flor_in_the_school = flor_in_the_school
+
+    def number_of_class(self):
+        return self.__number_of_class
+
+    def flor_in_the_school(self):
+        return self.__flor_in_the_school
+
+
+class ClassOfMath(Classes):
```
Comment (MarinaZelenska) [inline Oleksii_Fedorchuk/HW14_Oleksii_Fedorchuk/task1.py:34]: Why are all classes in 1 file? Could you please create each class in different module

File: Oleksii_Fedorchuk/HW14_Oleksii_Fedorchuk/task1.py [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    def director(self):
+        """ Encapsulation"""
+
+        return self.__director.capitalize()
+
+    @property
+    def secretary(self):
+        return self.__secretary
+
+    def number_of_students(self):
```
Comment (MarinaZelenska) [inline Oleksii_Fedorchuk/HW14_Oleksii_Fedorchuk/task1.py:24]: You are using @property selectively.

File: Oleksii_Fedorchuk/HW14_Oleksii_Fedorchuk/task1.py (hunk 1 above)
Comment (Oleksii-Fedorchuk) [reply Oleksii_Fedorchuk/HW14_Oleksii_Fedorchuk/task1.py]: Done.

File: Oleksii_Fedorchuk/HW14_Oleksii_Fedorchuk/task1.py (hunk 2 above)
Comment (Oleksii-Fedorchuk) [reply Oleksii_Fedorchuk/HW14_Oleksii_Fedorchuk/task1.py:34]: Done.

**Review by MarinaZelenska — APPROVED** [review]

**Pull request merged**